backend/backend/charge_rent.py

```diff
@@ -0,0 +1,45 @@
+"""Module enables users to be charged rent
+   when they land on owned properties
+"""
+
+import json
+import sys
+import cgitb
+from backend.player import Player
+from backend.game import get_games
+from backend.properties import Property
+
+cgitb.enable()
+
+
+def charge_rent(source=sys.stdin, output=sys.stdout):
+    """Entry point for a player to be charged
+       rent and property owner gains rent amount
+    """
+
+    output.write('Content-Type: application/json\n\n')
+    request = json.load(source)
+    player_id = request["player_id"]
+    games = get_games()
+    game_id = None
+
+    # Access the game the player is playing in
+    # and the position the player is on the board
+    with Player(player_id) as player:
+        position = player.board_position
+        for game in games:
+            if player.username in games[game]:
+                game_id = game
+                break
+
+        # Accesses owner of property; if the games's current
+        # turn player doesn't own it, charge the player and
+        # increase the property owner's balance
+        with Property(position, game_id) as propertie:
+            owner = propertie.owner
+            rent = propertie.rent
+            if owner.uid != player_id:
```

Comment:
Owner is just the player id, not a Player object, so you'll need to make another "with" before trying this.

Reply:
with Player(owner) as owner:
I think?

Reply:
In that case I can replace the 'owner.uid' to just 'owner' as I though the owner was a player object. Is that what you meant? All I need to do in regards to the owner is to get the id.

Reply:
Actually you're right, I had forgotten about the rent situation. Thanks for pointing out that the owner isn't a player object.

Reply:
Is it not possible to do what you suggested @gabystephanov by replacing `owner.uid` with just `owner`? The rent is related to the property and not the owner/player :-/